doc/tc-meetings/2015-06-10.md

```diff
@@ -0,0 +1,219 @@
+Julien: triaged issues, reviewed PRs. Started documenting Node.js’ release process and CI platform. Also started to document breaking changes between v0.12 and the converged repo between io.js and Node.js.
+Alexis: investigations on native addon WG.
+Bert: not a lot of technical work. Working on libuv changes to make the multi isolate/workers stuff work. PR #396 in libuv.
+Ben: reviewed PRs.
+Domenic: some reviews and tried to gear up for 3.0.
+Trevor: Lot of reviewing and landing of PRs, prototype for lower-level JS API. Worked on async listeners API.
+Bradley: here for rlimits discussion on the agenda.
+Mikeal: launch of the foundation, getting the PR ready and working on nodeconf that starts tomorrow.
+
+### Today’s agenda
+
+#### Add working group state per Project Lifecycle
+
+Mikeal: pretty much resolved, we just need to close the PR.
+
+#### Chrome 43 released; time for V8 4.3
+
+Domenic: not sure if there’s much progress on that. Haven’t seen news from nan.
+Bert: remove tsc-agenda, someone should put back the tag when there’s news.
+
+#### TSC needs to elect a board representative.
+
+Mikeal: thought that we had a resolution, and Bert and Rod would both be on the board for a short time.
+Bert: see if that can stay that way in the long term. Any objection?
+Michael: have we approached other people from the foundation to see if that’s possible?
+Bert: haven’t reached out yet. Who is organizing these meetings?
+Mikeal: Mike Dolan.
+Bert: Will send an email to see if that’s feasible.
+Michael: what about voting?
+Mikeal: we would need to put down just one person who has voting rights.
+Michael: having two present in meetings, only one who has the vote is fine by me.
+Bert: I don’t expect to be any kind of contention, trademark is probably the only one I can think of now.
+Mikeal: It seems that people don’t have a good idea of what will be decided during these meetings, so maybe we can just flip a coin for this short period of time, and then when we know more of what’s discussed in these meetings we can have a vote.
+Michael: maybe have some sort of alternance between two people in the beginning?
+Bert: I think it would still be good to have one person who will always be there.
+Bert: proposed resolution: let’s both (Rod and I) be there and then we’ll reevaluate later.
+
+#### Nominating new collaborators to this repository
+
+See https://github.com/joyent/node/issues/25481.
+
+Main question was: should we add nominated collaborators who are not io.js collaborators too the “collaborators” team in the nodejs organization too?
+
+Decision was that it might be confusing at first, so onboarding of these collaborators will be done first in joyent/node only, but they will conceptually be Node.js collaborators (as in collaborators in the nodejs GitHub organization). Shortly after they will be added as io.js collaborators by people familiar with the onboarding process at io.js.
+
+#### Node should not automatically change rlimits
```

Comment:
Did no one bring up my pre-meeting comment at https://github.com/nodejs/node/issues/53#issuecomment-110894244?

What was the conclusion of this?

Restating myself:

>I just want to make sure this gets heard about the rlimit thing: **the current behavior or equivalent must remain on OS X.**